### PR TITLE
Vulkan SDK version 1.1.77.0

### PR DESCRIPTION
**Pull request by apenngrace**
I updated this cask to remove a redundant folder (that matches the name of the file archive) in order to clean up the path to the resources of this Cask.

before : `/usr/local/Caskroom/vulkan-sdk/1.1.77.0/vulkansdk-macos-1.1.77.0/`
now: `/usr/local/Caskroom/vulkan-sdk/1.1.77.0/`

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/vulkan-sdk.rb [hunk 1]
```diff
@@ -8,17 +8,24 @@
 
   depends_on macos: '>= :el_capitan'
 
-  binary "#{staged_path}/vulkansdk-macos-#{version}/macOS/bin/vulkaninfo"
+  binary "#{staged_path}/macOS/bin/vulkaninfo"
+
+  # Move contents of redundant folder (that matches the name of the archive) up a folder
+  # and then delete that folder.
+  preflight do
+    system_command '/usr/bin/ditto', args: ['--', "#{staged_path}/vulkansdk-macos-#{version}", staged_path.to_s]
```
Comment (vitorgalvao) [inline Casks/vulkan-sdk.rb]: Why use `ditto` instead of moving?

Comment (vitorgalvao) [reply Casks/vulkan-sdk.rb]: Use ruby methods instead of calling system commands.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by apenngrace** [comment]
Ok, I changed the system_command calls to FileUtils calls per your comments.

File: Casks/vulkan-sdk.rb [hunk 2]
```diff
@@ -13,8 +13,8 @@
   # Move contents of redundant folder (that matches the name of the archive) up a folder
   # and then delete that folder.
   preflight do
-    system_command '/usr/bin/ditto', args: ['--', "#{staged_path}/vulkansdk-macos-#{version}", staged_path.to_s]
-    system_command '/bin/rm', args: ['-rf', "#{staged_path}/vulkansdk-macos-#{version}"]
+    FileUtils.mv Dir.glob("#{staged_path}/vulkansdk-macos-#{version}/*"), staged_path.to_s
+    FileUtils.remove_dir "#{staged_path}/vulkansdk-macos-#{version}", force: true
```
Comment (vitorgalvao) [inline Casks/vulkan-sdk.rb]: Why the `force`?

**Comment by apenngrace** [comment]
Let me know if anything else is needed before merging.  Thanks!

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**